app/components/indicators/AnimatedLinearProgressFull.jsx

```diff
@@ -52,7 +52,7 @@ const AnimatedLinearProgressFull = ({
       <div className={classNames("linear-progress", animationType)}>
         {getDaemonSynced || isSPV ? (
           <div
-            className={"linear-progress-bar " + (error ? "error" : null)}
+            className={classNames("linear-progress-bar", error && "error")}
```

Comment:
I am not a fan of this change, because if error is false it will become "linear-progress-bar false"